src/data/cryptography/index.ts

```diff
@@ -0,0 +1,5 @@
+import { ForgetPasswordTokenGenerator } from './forget-password-token-generation'
```

Comment:
why not just `export * from './forget-password-token-generation'`? All others index are this pattern

Reply:
I've already seen this implementation, but I don't usually use it, but I'm going to implement it because for this context that we have only one exported object, it makes perfect sense.